plugin/execute_command.py

```diff
@@ -14,27 +14,32 @@ class LspExecuteCommand(LspTextCommand):
     def __init__(self, view):
         super().__init__(view)
 
-    def run(self, edit, command_name=None, command_args=None) -> None:
+    def run(self, edit, command_name=None, command_args=None, modal_result=True) -> None:
```

Comment:
I like the idea of letting the user specify `modal_result`, but this functionality was meant to be separate from the code action command execution. Can we move it to another PR?